pkg/operator/ceph/cluster/mgr/config.go

```diff
@@ -43,30 +43,30 @@ type mgrConfig struct {
 }
 
 func (c *Cluster) dashboardPort() int {
-	if c.dashboard.Port == 0 {
+	if c.spec.Dashboard.Port == 0 {
 		// default port for HTTP/HTTPS
-		if c.dashboard.SSL {
+		if c.spec.Dashboard.SSL {
 			return dashboardPortHTTPS
 		} else {
 			return dashboardPortHTTP
 		}
 	}
 	// crd validates port >= 0
-	return c.dashboard.Port
+	return c.spec.Dashboard.Port
 }
 
 func (c *Cluster) generateKeyring(m *mgrConfig) (string, error) {
 	user := fmt.Sprintf("mgr.%s", m.DaemonID)
 	access := []string{"mon", "allow profile mgr", "mds", "allow *", "osd", "allow *"}
-	s := keyring.GetSecretStore(c.context, c.Namespace, &c.ownerRef)
+	s := keyring.GetSecretStore(c.context, c.clusterInfo, &c.clusterInfo.OwnerRef)
```

Comment:
ditto, just pass `clusterInfo`?

Reply:
`GetSecretStore()` actually needs to take the ownerRef separately because in some scenarios we get a secret store that has a different parent than the CephCluster.